pkg/skaffold/build/tag/git_commit.go

```diff
@@ -57,36 +53,44 @@ func (c *GitCommit) GenerateFullyQualifiedImageName(workingDir string, opts *Opt
 func generateNameGitShellOut(workingDir string, opts *Options) (string, error) {
```

Comment:
Now that we're adding some more logic to git-shell impl, are we always going to get the same result with git-shell vs. go-git? Do we care? Should we just drop the library?

Reply:
The same features should be implemented by both paths. Both paths are also tested. If that’s not the case, it’s an error

Reply:
SGTM